corehq/warehouse/management/commands/create_batch.py

```diff
@@ -1,52 +1,26 @@
 from __future__ import absolute_import
-from django.core.management import BaseCommand, CommandError
-from dimagi.utils.parsing import string_to_utc_datetime
+from datetime import date
+from django.core.management import BaseCommand
 from corehq.warehouse.models import Batch
 
 
-USAGE = """Usage: ./manage.py create_batch <batch_id> -s <start_datetime> -e <end_datetime>
+USAGE = """Usage: ./manage.py create_batch
 """
 
 
 class Command(BaseCommand):
     """
-    Example: ./manage.py create_batch 222617b9-8cf0-40a2-8462-7f872e1f1344 -s 2017-05-01 -e 2017-06-01
+    Example: ./manage.py create_batch
     """
     help = USAGE
 
-    def add_arguments(self, parser):
-        parser.add_argument('batch_id')
+    def handle(self, **options):
+        last_batch = Batch.objects.filter(completed_on__isnull=False).order_by('-end_datetime').first()
+        start = last_batch.end_datetime
+        end = date.today()
```

Comment:
Not knowing much about how this works - I assume it's fine if this ends up being quite a large window? We'd never want to do anything explicitly?

Reply:
we should allow windows to be any size - from a couple hours to a couple weeks, but we should be setting this to datetime.utcnow().

in order for this range method to work, the following needs to be true:

- we need to store utc timestamps in the batch table
- we need to be comparing to last modified utc timestamps in the source data when we pull, which are set from server processes (not any timestamps that come from the phones)
- we need to be comparing last_modified with > start_datetime and <= end_datetime